text/0002-extracting-necko.md

```diff
@@ -0,0 +1,67 @@
+# Problem
+
+Can we break out a Gecko component, such as Necko, for reuse in background services and on mobile platforms?
+
+## Motivation
+
+If — as we expect — Mozilla will need to build consistent native applications across platforms, and background services on desktop, it would be beneficial in theory to leverage our existing investment in our network layer, avoiding the confusing situation of having two ways to configure network settings, and avoiding the possibility of different behaviors in different parts of an application.
```

Comment:
Yes, we do. The mention of desktop background services in particular is an attempt to imply _without Gecko and a profile running_, which is the desktop equivalent to native mobile apps that don't have Gecko at all. I'll clarify.